configure

```diff
@@ -2178,13 +2191,19 @@ if test "$have_gcc" = yes ; then
 	append_var CXXFLAGS "-Wno-empty-body"
 	append_var CXXFLAGS "-fno-operator-names"
 elif test "$have_icc" = yes ; then
+	# ICC does not support pedantic, while GCC and clang do.
 	add_line_to_config_mk 'CXX_UPDATE_DEP_FLAG = -MMD -MF "$(*D)/$(DEPDIR)/$(*F).d" -MQ "$@" -MP'
 fi;
 
 #
 # Set status about C++11 mode
 #
-append_var CXXFLAGS "-std=c++11"
+append_var CXXFLAGS "-std=${std_variant}11"
+
+if test "$pedantic" = yes ; then
+	# We *do* want the 'long long' extension.
+	append_var CXXFLAGS "-pedantic -Wno-long-long"
```

Comment:
Is the `-Wno-long-long` still needed? The `long long` type is part  of the c++11 standard, so this is no longer an extension.

Reply:
Ah! I didn't check for this, just moved around the lines.
I can throw a test during the week to remove this.
Thanks!

Reply:
Done